contracts/multisig/src/state.rs

```diff
@@ -14,10 +14,11 @@ use crate::{
 #[cw_serde]
 pub struct Config {
     pub governance: Addr,
+    pub rewards_contract: Addr,
+    pub grace_period: u64,
```

Comment:
We probably eventually want some mechanism to modify this grace period (through governance), but we can solve that later on. Maybe leave a TODO comment saying we need to add a way to modify the grace period after instantiation?